tests/utils_tests/test_log.py

```diff
@@ -0,0 +1,131 @@
+import logging
```

Comment:
I think `log_response` was previously mostly tested by `tests.logging_tests.tests.HandlerLoggingTests`
I understand why this was put in utils but do you think we should move this to: `tests/logging_tests/test_log_response.py`?

The main reason I am suggesting this is `django/utils/log.py` includes things like `AdminEmailHandler` which is tested in `tests/logging_tests/tests.py`. So it feels like we'd be suggesting that all of those tests should move across. I feel like putting this in `logging_tests` keeps the tests for `django/utils/log.py` together

Reply:
Thank you for noticing this, you have a good point. I will move this and push some tweaks.